src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -48,6 +48,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Folder Include="JobHost\" />
```

Comment:
What is this for?

Reply:
Don't know how that got included. Removed it